parts/k8s/kubernetesmastervars.t

```diff
@@ -267,8 +266,8 @@
       "[concat(variables('masterFirstAddrPrefix'), add(3, int(variables('masterFirstAddrOctet4'))))]",
       "[concat(variables('masterFirstAddrPrefix'), add(4, int(variables('masterFirstAddrOctet4'))))]"
     ],
-    "masterEtcdServerPort": 2380,
-    "masterEtcdClientPort": 2379,
+    "masterEtcdServerPort": "[parameters('masterEtcdServerPort')]",
```

Comment:
Moved these vals to `pkg/acsengine/const.go` (and added appropriate changes elswhere).